### DISCUSSION TITLE
geplante Startzeit mitanzeigen

### DISCUSSION THREAD

**Discussion by sirius-black-2** [camueller/SmartApplianceEnabler#438] [Allgemein]
Eine kurze Anregung als "nice to have". Im Sunny Portal wird unter "Aktueller Status und Prognose"  auch angezeigt, wann die Verbraucher eingeplant werden. Ist es möglich diese Information auch auf der SAE Startseite unter dem entsprechenden Verbraucher anzuzeigen?
Beste Grüße und alle, die an diesem Projekt mitarbeiten, insbesondere Axel: Vielen Dank für Eure Arbeit!
![Prognose](https://user-images.githubusercontent.com/99977252/210997478-fc2ff07e-8fad-4015-9b2a-a4a84005fa26.PNG)

**Comment by camueller** [comment]
Die Einplanung ist Sache des SHM und dieser übergibt aus Schaltbefehlen keine anderen Daten an den SAE. Insofern ist es nicht möglich, solche Daten im SAE anzuzeigen.
